blueocean-pipeline-editor/pom.xml

```diff
@@ -72,6 +72,11 @@
                     <groupId>com.squareup.okhttp3</groupId>
                     <artifactId>okhttp</artifactId>
                 </exclusion>
+                <!-- commons-lang3 is provided by commons-lang3-api plugin -->
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-lang3</artifactId>
+                </exclusion>
```

Comment:
can be removed when this is merged and released:
- https://github.com/jenkinsci/sonarqube-plugin/pull/13